src/style.css

```diff
@@ -1,3 +1,67 @@
+/* stylelint-disable font-family-name-quotes */
+/* stylelint-disable rule-empty-line-before */
+
+:root {
+  --main-color: #00acdc;
```

Comment:
Consider using a CSS variable for the blue color used in the `.item-link:hover` and `.is-active` classes to enhance maintainability and readability. This will ensure consistency and make it easier to update the color in the future.